### PR TITLE
Latest Changes

### PR DESCRIPTION
**Pull request by OrsiPapp**
A lot of changes have been made. These are all changes we made today in our class and after class. This includes us creating a register user page and login page, and style changes. I worked a few hours on the style after class, especially because of an issue with the background image.

Issues:

There is an issue on the FAQ page. The questions do not roll out. I renamed collapsible to collapsible_faq which is causing the issue, but I cannot find it. It can be in faq.js, faq.php or style.css.

The background design does not show throughout the website and it needs to be adjusted so that the coffee beans are behind everythig.

No other major issues to report. I will continue to work on it perhaps tomorrow from around 6 PM, but just in case you will start to work on the project before I would get to it, please pay attention to these issues and feel free to correct them. If you do so, please let me know what the issue was. You can do that on GitHub while doing your review.

### CODE REVIEW

**Review by OrsiPapp — COMMENTED** [review]
Please be aware of issues with the FAQ page. I am kindly asking for help with that.

File: contact.php [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Pacifico&display=swap" rel="stylesheet">
```
Comment (habibmohammed) [inline contact.php:6]: Should be removed

Comment (OrsiPapp) [reply contact.php:6]: Okay!

File: retrieve-time.php [hunk 2]
```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html>
   <head>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Pacifico&display=swap" rel="stylesheet">
```
[the remark this answers is not in the document]
Comment (OrsiPapp) [reply retrieve-time.php:6]: Sounds good!

File: faq.php [hunk 3]
```diff
@@ -35,8 +38,8 @@
         </div>
       <hr>
 
-      <button type="button" class="collapsible">Who's our coffee supplier?</button>
-        <div class="collapsible_div">
```
Comment (habibmohammed) [inline faq.php:39]: Same here

**Pull request merged**